Client/Frontend/Browser/Onboarding/OnboardingAdsCountdownView.swift

```diff
@@ -112,12 +113,13 @@ extension OnboardingAdsCountdownViewController {
             mainStackView.snp.makeConstraints {
                 $0.leading.equalTo(self.safeArea.leading)
                 $0.trailing.equalTo(self.safeArea.trailing)
-                $0.bottom.equalTo(self.safeArea.bottom)
+                $0.bottom.equalTo(snp.bottom)
             }
             
             descriptionView.addSubview(descriptionStackView)
             descriptionStackView.snp.makeConstraints {
-                $0.edges.equalToSuperview().inset(UX.descriptionContentInset)
+                $0.top.leading.trailing.equalToSuperview().inset(UX.descriptionContentInset)
```

Comment:
and this would change to:

```
descriptionStackView.snp.makeConstraints {
    $0.edges.equalTo(descriptionView.safeArea.edges).inset(UX.descriptionContentInset)
}
```

Now you don't have to worry about "52" :). It will automatically calculate how far from the bottom it should be.